transports/circuit-relay-v2/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    prost_build::compile_protos(&["src/proto/message.proto"], &["src/proto"])
+        .unwrap();
+}
```

Comment:
Just a quick FYI, we should not use prost but instead use `quick-protobuf` :). See https://github.com/libp2p/rust-libp2p/issues/3024

Reply:
Got it. Thanks for catching that.